lldb/source/Plugins/LanguageRuntime/Swift/SwiftLanguageRuntime.cpp

```diff
@@ -608,6 +613,51 @@ bool SwiftLanguageRuntimeImpl::AddJitObjectFileToReflectionContext(
       });
 }
 
+bool SwiftLanguageRuntimeImpl::AddObjectFileToReflectionContext(
+    ModuleSP module, ObjectFile &obj_file, llvm::Triple::ObjectFormatType obj_format_type) {
+  auto obj_file_format = GetObjectFileFormat(obj_format_type);
+
+  auto &memory_reader = m_reflection_ctx->getReader();
+  auto *lldb_memory_reader = (LLDBMemoryReader *)&memory_reader; 
```

Comment:
You could avoid the cast by calling GetMemoryReader() and asserting that it's the same one.